enricher/ecs/ecs.go

```diff
@@ -44,24 +46,28 @@ func NewEnricher() *Enricher {
 	}
 
 	return &Enricher{
-		canvaAWSAccount: os.Getenv("CANVA_AWS_ACCOUNT"),
-		canvaAppName:    os.Getenv("CANVA_APP_NAME"),
-		logGroup:        os.Getenv("LOG_GROUP"),
-		ecsTaskFamily:   ecsTaskFamily,
-		ecsTaskRevision: ecsTaskRevision,
+		canvaAWSAccount:    os.Getenv("CANVA_AWS_ACCOUNT"),
+		canvaApplicationId: os.Getenv("CANVA_APP_ID"),
```

Comment:
Needs to be updated to reflect change in https://github.com/Canva/canva/pull/507179